xclim/indices/_threshold.py

```diff
@@ -1,17 +1,12 @@
-import logging
-
 import numpy as np
-import xarray as xr
```

Comment:
Why is the shorthand xr not ok ?

Reply:
Both perfectly fine. Matter of preference, I suppose. I'll avoid making these kinds of changes in the future.